db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
+])
+
+User.create!([
+  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
+])
+
+Resource.create!([
+  {name: "Dining Room", community_id: 1, default_calendar_view: "week", meal_abbrv: "CH", hidden: false}
+])
+
+
+
```

Comment:
I'm about to check in this whole batch.  If the new fake_user_data.rake file comes out wrong, let me know.

FWIW, the editor I'm using is Gedit, the one that's bundled with Ubuntu.